### PR TITLE
Added details flag to apiapp list and apiapp show

### PR DESCRIPTION
**Pull request by christav**
Added -d switch for details on list command, added apiapp show command, rerecorded mocks.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@christav__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ctavares). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: lib/commands/arm/apiapp/apiapp._js [hunk 1]
```diff
@@ -36,28 +38,39 @@ exports.init = function initApiAppCommands(cli) {
   apiapp.command('list [resource-group]')
     .description($('List currently deployed ApiApps in a subscription or resource group'))
     .option('-g --resource-group <resource-group>', $('the resource group name, if not given list all ApiApps in subscription'))
-    .option('-s --subscription <subscription>', $('The subscription identifier.'))
+    .option('-s --subscription <subscription>', $('The subscription identifier'))
+    .option('-d --detailed', $('Include ApiApp package version and auth setting (call will be slower)'))
```
Comment (markcowl) [inline lib/commands/arm/apiapp/apiapp._js]: does it make sense to use -v for this (for parallelism with other commands)?

**Comment by markcowl** [comment]
one question, otherwise LGTM

File: lib/commands/arm/apiapp/apiapp._js (hunk 1 above)
Comment (christav) [inline lib/commands/arm/apiapp/apiapp._js]: Lots of other commands use -d for this - account show, group list, all the app insights commands, tag list, and a bunch more.

**Pull request merged**